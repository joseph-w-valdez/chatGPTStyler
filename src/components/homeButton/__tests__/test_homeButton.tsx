import * as React from "react";
import { HomeButton } from "../HomeButton";
import renderer from "react-test-renderer";

it("component renders", () => {
    const dataTestid = "test-id";
    const btnLabel = "click me uWu";

    expect(typeof dataTestid).toBe("string");
    expect(typeof btnLabel).toBe("string");

    const tree = renderer
        .create(
            <HomeButton
                dataTestid={dataTestid}
                onClick={jest.fn()}
                btnLabel={btnLabel}
            />,
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
