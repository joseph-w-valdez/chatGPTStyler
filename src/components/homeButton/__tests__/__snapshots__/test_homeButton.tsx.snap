// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<button
  className="btn"
  data-testid={Any<String>}
  onClick={[Function]}
>
  {Any<String>}
</button>
`;
